style.css

```diff
@@ -3,28 +3,41 @@ body
   margin: 0;
   padding: 0;
   background: #FFFFFF;
-  position: relative;
+  font-family: 'Roboto', sans-serif;
+  font-size: 13px;
+  width: 100%;
+}
+
+.logoPicture
+{
+	width: 9%;
+	padding-right: 20px;
 }
 
 nav
 {
 	border-bottom: 1px solid #d9dadb;
+	justify-content: space-around;
 }
 
 nav a
 {
-	color: #686868;
+	color: #636262;
+	width: 10%;
+	padding-right: 40px;
+	padding-left: 40px;
+	/*float: left;*/
+	white-space: nowrap;
```

Comment:
It prevents text from wrapping to another line. I was styling the menu bar when I saw that the "new evaluation" button wrap when resizing the window so i thought that i should stop it.